### PR TITLE
[PAPI-278] New vault transaction quotes

### PR DESCRIPTION
**Pull request by drmathias**
New vault endpoints to quote:

* Proposing to create a certificate
* Proposing to revoke a certificate
* Proposing to change the minimum pledge amount
* Proposing to change the minimum vote amount
* Pledging to a proposal
* Withdrawing a pledge to a proposal
* Voting on a proposal
* Withdrawing a vote on a proposal
* Completing a vault proposal
* Redeeming a certificate

image

### CODE REVIEW

File: src/Opdex.Platform.Application.Abstractions/EntryCommands/VaultGovernances/CreateVaultProposalCreateCertificateQuoteCommand.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+/// Command to quote proposing the creation of a vault certificate.
+/// </summary>
+public class CreateVaultProposalCreateCertificateQuoteCommand : BaseTransactionQuoteCommand
+{
+    /// <summary>
+    /// Creates a command to quote proposing the creation of a vault certificate.
+    /// </summary>
+    /// <param name="vault">Address of the vault.</param>
+    /// <param name="owner">Address of the certificate owner.</param>
+    /// <param name="amount">Proposed minimum pledge amount.</param>
```
Comment (mrtpain) [inline src/Opdex.Platform.Application.Abstractions/EntryCommands/VaultGovernances/CreateVaultProposalCreateCertificateQuoteCommand.cs:18]: minor comment change, amount is the number of tokens for the cert

File: src/Opdex.Platform.Application.Abstractions/EntryCommands/VaultGovernances/CreateVaultProposalPledgeQuoteCommand.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+/// </summary>
+public class CreateVaultProposalPledgeQuoteCommand : BaseTransactionQuoteCommand
+{
+    /// <summary>
+    /// Creates a command to quote voting on a vault proposal.
+    /// </summary>
+    /// <param name="vault">Address of the vault.</param>
+    /// <param name="proposalId">Id of the proposal in the vault.</param>
+    /// <param name="amount">Amount of CRS to vote.</param>
+    /// <param name="inFavor">If the vote is in favor of the proposal.</param>
```
Comment (mrtpain) [inline src/Opdex.Platform.Application.Abstractions/EntryCommands/VaultGovernances/CreateVaultProposalPledgeQuoteCommand.cs:18]: minor, change vote language to pledge

**Review by mrtpain — APPROVED** [review]
Only a couple comments, lgtm 👍

File: src/Opdex.Platform.Application.Abstractions/EntryCommands/VaultGovernances/CreateVaultProposalPledgeQuoteCommand.cs (hunk 2 above)
Comment (drmathias) [reply src/Opdex.Platform.Application.Abstractions/EntryCommands/VaultGovernances/CreateVaultProposalPledgeQuoteCommand.cs]: Sorted

**Pull request merged**